frontend/src-tauri/src/store.rs

```diff
@@ -45,6 +47,26 @@ pub fn get_realclose(app_handle: &AppHandle) -> bool {
         }
     }
 }
+pub fn get_minimizeonstartup(app_handle: &AppHandle) -> bool {
+    let minimizeonstartup = with_store(
+        app_handle.clone(),
+        app_handle.state(),
+        ".settings.json",
+        |store| {
+            return Ok(store
+                .get("minimizeonstartup")
+                .and_then(|json| json.as_bool()));
+        },
+    );
+
+    match minimizeonstartup {
+        Ok(r) => r.unwrap_or(false),
```

Comment:
should this return True by default as your intention was to minimize by default?

Reply:
see my general comment too